.azure/pipelines/devBuilds.yml

```diff
@@ -30,3 +30,28 @@ stages:
         path: artifacts/log/
         publishOnError: true
         includeForks: true
+
+- stage: build_servers
+  displayName: Build Servers
+  jobs:
+  # Build servers on Windows (x64)
+  - template: jobs/default-build.yml
+    parameters:
+      codeSign: false
+      jobName: Windows_build
+      jobDisplayName: "Build: Servers"
+      agentOs: Windows
+      steps:
+      - script: git submodule init
+      - script: git submodule update --recursive
+      - script: cd ./src/Servers
+      - script: ./build.cmd
+                -ci
+                -arch x64
+                /bl:artifacts/log/build.x64.binlog
+        displayName: Build x64
+      artifacts:
+      - name: Windows_Logs
```

Comment:
Rename both the servers and components versions of to have a different name. Otherwise, I think they can overwrite other runs.